examples/factors/README.md

```diff
@@ -93,25 +93,27 @@ Use this command any time you'd like to check your progress.
 
 Our next objective is to provide the guest program with input. Before we implement this, let's take a closer look at how we run and prove the guest program in `factors/src/main.rs`.
 
-In the starter template project, our host driver program creates an executor environment before constructing an executor.  When `Executor::run()` is called, it will produce a session. Calling `Session::prove()` will cause our guest program to execute, producing a receipt:
+In the starter template project, our host driver program creates an executor environment before constructing a prover.  When `Prover::prove_elf()` is called, it will produce a receipt:
 
 ```rust
     use factors_methods::{MULTIPLY_ELF, MULTIPLY_ID};
     use risc0_zkvm::{
-      default_executor_from_elf,
+      default_prover,
       serde::{from_slice, to_vec},
       ExecutorEnv,
     };
 
     // First, we construct an executor environment
     let env = ExecutorEnv::builder().build().unwrap();
 
-    // Next, we make an executor, loading the (renamed) ELF binary.
-    let mut exec = default_executor_from_elf(env, MULTIPLY_ELF).unwrap();
-
     // We're not quite ready to run these steps yet
```

Comment:
We don't need to resolve this on this PR, but I'm wondering whether we should strike `we're not quite ready to run these steps yet`. I don't really follow what this comment is about.

Reply:
It's part of the tutorial, we should probably do a deeper review of the entire README to make sure it still flows correctly. It's possible that certain steps aren't really needed anymore.

Reply:
Agreed with both "do a deeper review" and "don't need to resolve on this PR"